src/style.css

```diff
@@ -1,3 +1,81 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
 body {
   margin: 0;
 }
+
+header {
```

Comment:
It's not recommended to style HTML elements directly (except for `html` and `body`). Instead, you should add a class to the `header` element and style that class. (CSS checklist)